### PR TITLE
[DD-452] Jekyll include for the section header + illustration + paragraph

### PR DESCRIPTION
**Pull request by tshen815**
[Ticket](https://circleci.atlassian.net/browse/DD-452)
[Preview](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/DD-452/getting-started-experiment-header-preview/2.0/concepts/?force-all)
**NOTE:** The preview link is just demoing the section header on the concepts page as we currently dont have the layout in place for the Getting Started experiment yet. 

# Changes

- add includes HTML layout for section header
- add styles for section header
- import SVGs for header banner image

# Description
We need to create an include section that we can reuse for each section for the Getting Started experiment. The includes layout should accept the title, banner image and intro paragraph as per the design. The includes should also be responsive to window size changes. 

# Considerations
This PR is a part of the [Getting Started experiment epic](https://circleci.atlassian.net/browse/DD-449). The other associated PRs: #6564 & #6544 

# Screenshots 
**Desktop:**
Screen Shot 2022-03-21 at 2 16 59 PM

**Mobile:** 
Screen Shot 2022-03-21 at 2 17 15 PM

### CODE REVIEW

File: jekyll/_includes/getting-started-section-header.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class='section-header-wrapper'>
+  <h2 class='section-title'>{{include.title}}</h2>
+  <img class='image-banner' src="{{include.imagePath}}" alt="header banner" />
+  <div class='content-wrapper'>
+    <p>
+      {{include.content}}
+    </p>
+  </div>
```
Comment (rsylvian) [inline jekyll/_includes/getting-started-section-header.html:8]: if content could be anything, I don't think it should be in a `<p>`?

File: src/styles/includes/_getting-started-section-header.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.section-title {
+  font-family: 'Roboto';
+  font-style: normal;
```
Comment (rsylvian) [inline src/styles/includes/_getting-started-section-header.scss:3]: can we apply this directly to `.section-header-wrapper` instead?

Comment (tshen815) [reply src/styles/includes/_getting-started-section-header.scss:3]: good call, Ill nest the styling as well

File: jekyll/_includes/getting-started-section-header.html (hunk 1 above)
Comment (tshen815) [reply jekyll/_includes/getting-started-section-header.html:8]: good call, I can update it to a `<span>`?

**Review by chantelle11 — COMMENTED** [review]
I am not sure when this will be merged in considering it goes with the jekyll 2 by 2 grid but the ending is a little abrupt imo right now

Screen Shot 2022-03-22 at 3 41 18 PM

File: jekyll/assets/img/docs/quick-start--first-step.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<path d="M365 144C285.471 144 221 79.529 221 0" stroke="#04AA51" stroke-width="4"/>
+<path d="M365 90C315.294 90 275 49.7056 275 0" stroke="#04AA51" stroke-width="4"/>
+<path d="M365 36C345.118 36 329 19.8823 329 0" stroke="#04AA51" stroke-width="4"/>
+<path d="M731 36C810.529 36 875 100.471 875 180" stroke="#003740" stroke-width="4"/>
+<path d="M731 90C780.706 90 821 130.294 821 180" stroke="#003740" stroke-width="4"/>
+<path d="M731 144C750.882 144 767 160.118 767 180" stroke="#003740" stroke-width="4"/>
+<path d="M365 36H731" stroke="#C7E8D7" stroke-width="4"/>
+<path d="M365 90H731" stroke="#C7E8D7" stroke-width="4"/>
+<path d="M365 144H731" stroke="#C7E8D7" stroke-width="4"/>
+<circle cx="321" cy="44" r="19" fill="#04AA51"/>
```
Comment (chantelle11) [inline jekyll/assets/img/docs/quick-start--first-step.svg:14]: I am not sure if this is possible but it seems like the svg's only differ on this 1 line. is there a way to separate it and combine it to reuse the background but have the ball moving?

(Might not be worth the time effort just a thought to consider)

**Review by chantelle11 — APPROVED** [review]
The demo works great! Can you include an example on how to use `getting-started-section-header`

**Review by teesloane — APPROVED** [review]
Looks great, nice work!

File: jekyll/assets/img/docs/quick-start--first-step.svg (hunk 3 above)
Comment (tshen815) [reply jekyll/assets/img/docs/quick-start--first-step.svg:14]: Great call out, maybe we could look into that if the experiment is a success?

**Pull request merged**